docs-markdown/src/test/suite/index-coverage.ts

```diff
@@ -34,14 +34,12 @@ function setupNyc() {
 }
 
 export function run(): Promise<void> {
-	let nyc = setupNyc();
+	// tslint:disable-next-line
+	let nyc = setupNyc(); // "const" causes coverage report error
+
 	// Create the mocha test
 	const mocha = new Mocha({
 		color: true,
-		reporter: "mocha-junit-reporter",
```

Comment:
I didnt notice any problems with the mocha-junit-reporter? That is how we get the reported problems with which tests had failed in CI. Do we actually want to remove this?

Reply:
ahh, got it. I wanted to see the output on the console. Reverting for now. Maybe we can add both later.